local/common_vars

```diff
@@ -0,0 +1,9 @@
+AWS_SSH_KEY_NAME="${AWS_SSH_KEY_NAME:-splice}"
+export AWS_SSH_KEY_NAME
+
+export CLUSTER_IP="$(ifconfig docker0 | grep inet | awk '{print $2}' | head -1)"
+#export CLUSTER_IP="172.17.0.1"
```

Comment:
Should we add another variable in EXTRA_VARS to ensure we don't attempt to create lets_encrypt certs for local running?